spec/lib/pre_assembly/bundle_spec.rb

```diff
@@ -206,28 +218,14 @@
   end
 
   describe '#objects_to_process' do
-    it "has the correct list of objects to re-accession if specified with only option" do
-      b = described_class.new(context_from_proj(:proj_sohp3))
```

Comment:
wonderful descriptive names for these projects -- why not  bundle_e_files_only.yaml?   🙄

Reply:
#262